simpleclient/src/test/java/io/prometheus/client/SummaryTest.java

```diff
@@ -134,55 +219,69 @@ public Integer call() {
     assertEquals(123, result);
 
     Summary.Timer timer = noLabels.startTimer();
-    elapsed = timer.observeDuration();
+    double elapsed2 = timer.observeDuration();
+    assertEquals(10, elapsed2, .001);
+
     assertEquals(3, getCount(), .001);
     assertEquals(30, getSum(), .001);
-    assertEquals(10, elapsed, .001);
   }
 
   @Test
-  public void noLabelsDefaultZeroValue() {
+  public void testNoLabels() {
     assertEquals(0.0, getCount(), .001);
     assertEquals(0.0, getSum(), .001);
-  }
+    assertEquals(null, getMin());
+    assertEquals(null, getMax());
 
-  private Double getLabelsCount(String labelValue) {
```

Comment:
Please don't change this. It makes the diff larger than it needs to be, and is also harder to understand.